lua/leap-by-word.lua

```diff
@@ -48,39 +96,22 @@ local function get_targets(winid, char, direction)
       lnum = fold_end + 1
     else
       local line = vim.fn.getline(lnum)
+      local chars = vim.fn.split(line, '\\zs\\ze')
```

Comment:
what is the `\\zs\\ze` here?

Reply:
From doc on `split()`:

> To split a string in individual characters:  
> :for c in split(mystring, '\zs')

It handles multibyte characters and considers combining characters as part of one character.

Not sure why I used `\zs\ze`. `\zs` on its own also works.